extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js

```diff
@@ -1225,7 +1229,7 @@
 				}
 			}
 
-			if (_t.hasValue) {
+			if (_t.valueNode !== null && _t.hasValue) {
```

Comment:
We have the same test in ControlLink and ControlSelection.
Do you think we should suppress this test in the 3 cases ?